types/google.visualization/index.d.ts

```diff
@@ -1196,16 +1196,23 @@ declare namespace google {
             backgroundColor?: string | ChartStrokeFill;
             colors?: string[];
             enableInteractivity?: boolean;
+            fontName?: string;
+            fontSize?: number;
             forceIFrame?: boolean;
             height?: number;
             timeline?: {
                 barLabelStyle?: LabelStyle;
                 colorByRowLabel?: boolean;
                 groupByRowLabel?: boolean;
-                rowLabelStyle?: LabelStyle;
+                rowLabelStyle?: LabelStyle | null;
+                showBarLabels?: boolean;
                 showRowLabels?: boolean;
-                singleColor?: string;
-            }
+                singleColor?: string | null;
+            };
+            tooltip?: {
+                isHtml?: boolean;
+                trigger?: 'focus' | 'none'
```

Comment:
I tried to check if we could re-use `ChartTooltip` here.
It turns out that it is partially supported:
* `textStyle` is supported, though it is not documented - _we should add it_
* `showColorCode` and `ignoreBounds`  are not supported
https://codepen.io/danmana/pen/ZEBOper?editors=1010
```suggestion
                trigger?: 'focus' | 'none';
                textStyle?: ChartTextStyle;
```

Reply:
@danmana Thanks for your review. MR is updated.